src/analysis/special_functions/stirling.lean

```diff
@@ -0,0 +1,237 @@
+/-
+Copyright (c) 2022. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Firsching, Fabian Kruse, Nikolas Kuhn
+-/
+import analysis.p_series
+import analysis.special_functions.log.deriv
+
+/-!
+# Stirling's formula
+
+This file proves Theorem 90 from the [100 Theorem List] <https://www.cs.ru.nl/~freek/100/>.
+It states that $n!$ grows asymptotically like $\sqrt{2\pi n}(\frac{n}{e})^n$.
+TODO: Add Part 2 to complete the proof
+
+## Proof outline
+
+The proof follows: <https://proofwiki.org/wiki/Stirling%27s_Formula>.
+
+### Part 1
+We consider the fraction sequence $a_n$ of fractions $\frac{n!}{\sqrt{2n}(\frac{n}{e})^n}$ and
+prove that this sequence converges against a real, positive number $a$. For this the two main
+ingredients are
+ - taking the logarithm of the sequence and
+ - use the series expansion of $\log(1 + x)$.
+-/
+
+open_locale topological_space big_operators
+open finset filter nat real
+
+namespace stirling
```

Comment:
I don't think you want that namespace, otherwise all your lemmas have to referenced by `stirling.stirling_seq_foo`